requirements.txt

```diff
@@ -8,3 +8,4 @@ loguru==0.7.2
 fastapi==0.111.0
 pydantic==2.8.2
 uvicorn[standard]==0.30.1
+numpy==1.26.4
```

Comment:
```
A module that was compiled using NumPy 1.x cannot be run in
NumPy 2.0.0 as it may crash. To support both 1.x and 2.x
versions of NumPy, modules must be compiled with NumPy 2.0.
Some module may need to rebuild instead e.g. with 'pybind11>=2.12'.

If you are a user of the module, the easiest solution will be to
downgrade to 'numpy<2' or try to upgrade the affected module.
We expect that some modules will need time to support NumPy 2.
```